### PR TITLE
chore: changelog updates and utils

### PR DESCRIPTION
**Pull request by r4mmer**
## Acceptance criteria

- Update changelog to https://keepachangelog.com/en/1.0.0/ standard
- Util updates and debugging logs

### CODE REVIEW

**Review by tuliomir — APPROVED** [review]

**Review by tuliomir — APPROVED** [review]

File: tests/qa.py [hunk 1]
```diff
@@ -57,12 +89,13 @@ def test_qa_sign_tx_with_token(cmd):
...
+            TxOutput(1, fake_script(), 1, True),
+            TxOutput(2, fake_script(), 1, True),
             TxOutput(fake.pyint(1), fake_script()),
             ]
-    tx = Transaction(1, [token.uid], inputs, outputs)
-    print("QA::sign_tx_with_token::token:", str(token))
-    print("QA::sign_tx_with_token::tx:", str(tx))
+    tx = Transaction(0, 1, [token.uid], inputs, outputs)
+    print("QA::sign_tx_with_aauthority::token:", str(token))
+    print("QA::sign_tx_with_aauthority::tx:", str(tx))
```
Comment (pedroferreira1) [inline tests/qa.py]:
```suggestion
    print("QA::sign_tx_with_authority::token:", str(token))
    print("QA::sign_tx_with_authority::tx:", str(tx))
```

File: tests/qa.py [hunk 2]
```diff
@@ -57,12 +89,13 @@ def test_qa_sign_tx_with_token(cmd):
...
     outputs = [
-            TxOutput(fake.pyint(1), fake_script(), 1),
+            TxOutput(1, fake_script(), 1, True),
+            TxOutput(2, fake_script(), 1, True),
             TxOutput(fake.pyint(1), fake_script()),
             ]
-    tx = Transaction(1, [token.uid], inputs, outputs)
-    print("QA::sign_tx_with_token::token:", str(token))
-    print("QA::sign_tx_with_token::tx:", str(tx))
+    tx = Transaction(0, 1, [token.uid], inputs, outputs)
```
Comment (pedroferreira1) [inline tests/qa.py:96]: What's this new first parameter?

**Review by pedroferreira1 — APPROVED** [review]

File: tests/qa.py (hunk 2 above)
Comment (r4mmer) [reply tests/qa.py:96]: This qa document was not updated on #67 where signal_bits was introduced so it was updated here.

The idea of the qa would be a form of manual validation for the UI, so the qa assignee can check that everything is being shown as intended on the screen.

File: tests/utils.py [hunk 3]
```diff
@@ -32,8 +32,8 @@ def fake_tx(
     outputs: Optional[TxOutput] = None,
     tokens: Optional[bytes] = None,
 ) -> Transaction:
-    tx_inps = inputs or [fake_input() for _ in range(fake.pyint(1, 10))]
-    tx_outps = outputs or [fake_output() for _ in range(fake.pyint(1, 10))]
+    tx_inps = inputs or [fake_input() for _ in range(fake.pyint(1, 5))]
```
Comment (msbrogli) [inline tests/utils.py:35]: Why are you reducing the number of inputs and outputs?

**Review by msbrogli — APPROVED** [review]

File: tests/utils.py (hunk 3 above)
Comment (r4mmer) [reply tests/utils.py:35]: When debugging some issues that were solved in another PR I needed to decrease the number of inputs and outputs so I could manually debug some issues.

**Pull request merged**